modules/hosted-che/partials/proc_providing-a-devfile.adoc

```diff
@@ -5,7 +5,7 @@
 [id="providing-a-devfile_{context}"]
 = Providing a devfile
 
-Providing a devfile in the root of the repository is recommended to define the development environment of the workspace created by the factory URL.
+Providing a link:https://devfile.io/[devfile] in the root of the repository is recommended to define the development environment of the workspace created by the factory URL.
```

Comment:
Providing a link: https://devfile.io/[devfile] in the root directory of the repository is recommended to define the development environment of the workspace created by the factory URL.

Reply:
this is adoc syntax - https://docs.asciidoctor.org/asciidoc/latest/macros/link-macro/

Reply:
ignore the "link:" part. Please reword the remainder of the sentence as suggested.

Reply:
```suggestion
Providing a link:https://devfile.io/[devfile] in the root directory of the repository is recommended to define the development environment of the workspace created by the factory URL.
```